utils/roles/update_synclist/vars/main.yml

```diff
@@ -0,0 +1,25 @@
+#  Copyright 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+---
+
+# Usage: main.yml
+get_nodes_python_script: "{{ role_path }}/files/get_nodes_from_inventory.py"
+update_dp_python_script: "{{ role_path }}/files/update_omniadb.py"
```

Comment:
it looks a typo of update_db_py...

Reply:
It is not typo, variable name is different and python file name is different